app.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-require('newrelic');
+if(process.env.NODE_ENV === 'production') require('newrelic');
```

Comment:
Hm. I just remembered that `NODE_ENV` is set to `production` during testing so that we can test the Google Analytics code.

Maybe we should *not* set `NODE_ENV` to `production` during testing, but instead check whether we are testing before adding the GA code.